src/ROUTES.ts

```diff
@@ -576,9 +576,13 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/tags/edit',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/tags/edit` as const,
     },
+    WORKSPACE_TAG_EDIT: {
+        route: 'settings/workspace/:policyID/tag/:tagName/edit',
+        getRoute: (policyID: string, tagName: string) => `settings/workspace/${policyID}/tag/${decodeURIComponent(tagName)}/edit` as const,
+    },
     WORKSPACE_TAG_SETTINGS: {
         route: 'settings/workspaces/:policyID/tag/:tagName',
-        getRoute: (policyID: string, tagName: string) => `settings/workspaces/${policyID}/tag/${encodeURIComponent(tagName)}` as const,
+        getRoute: (policyID: string, tagName: string) => `settings/workspaces/${policyID}/tag/${decodeURIComponent(tagName)}` as const,
```

Comment:
This seems a bit off. Should we be encoding the tag name here and decoding it in the linkingConfig params?

Reply:
Hm, actually makes sense

Reply:
Yea, the route gets messed up
![Screenshot 2024-03-19 at 9 18 16 AM](https://github.com/Expensify/App/assets/22219519/09240dff-6d8b-4b44-b337-644f4bb5ab7c)